fhir-term/src/main/java/com/ibm/fhir/term/spi/FHIRTermServiceProvider.java

```diff
@@ -67,6 +68,22 @@
      */
     boolean validateCode(Coding coding);
 
+    /**
+     * Indicates whether the given code is a member of the specified value set
+     *
+     * @param valueSet
+     *     the value set
```

Comment:
an external contract about whether the valueset must be pre-expanded?

Reply:
Added an `@apiNote` comment to the Javadoc